tests/integration/test_cli.py

```diff
@@ -20,24 +20,24 @@ def runner():
     return CliRunner()
 
 
-def test_pipeline_adds_request(runner, mongo_port, cfg, apache_req):
+def test_pipeline_adds_request(runner, mongo_uri, cfg, apache_req):
     with open(cfg) as fp:
         config = yaml.load(fp)
-    config['MONGO_CONNECTION'] = 'mongodb://localhost:%d' % mongo_port
+    config['MONGO_CONNECTION'] = [mongo_uri]
```

Comment:
Ah crap. I rebased, saw the conflict, and chose the wrong one. Thanks for catching that.